### PR TITLE
Style font sizes

### PR DESCRIPTION
**Pull request by bhajneet**
Mobile render

Home page:
image 1

Example page:
image 2

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -4,16 +4,3 @@
...
-}
-
-::-webkit-scrollbar-track {
-  /* -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3); */
-  background-color: #f6f0e4;
-}
-
-::-webkit-scrollbar-thumb {
-  background-color: #db5c87;
-}
```
Comment (Harjot1Singh) [inline frontend/src/App.css]: Missing newline

File: frontend/src/components/Home.css [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
   font-family: 'Open Sans', sans-serif;
-  font-size: 3.5em;
+  font-size: 1em;
 }
 
 .home .introduction .links {
   display: flex;
   flex-wrap: wrap;
-  width: 75%;
+  width: 83.333%;
```
Comment (Harjot1Singh) [inline frontend/src/components/Home.css]: Where are these magic numbers coming from lol

File: frontend/src/components/Home.css [hunk 3]
```diff
@@ -41,59 +41,66 @@
...
+  padding: .570em .755em;
+  margin: .570em;
+  border-radius: 0.43em;
 }
 
 .home .introduction .links .button:hover {
-  font-weight: bold;
-  background-color: rgba(189, 184, 174, 0.9);
+  background-color: #693E4B;
+  border-bottom-color: rgba(0,0,0,1);
```
Comment (Harjot1Singh) [inline frontend/src/components/Home.css]:
```suggestion
  border-bottom-color: black;
```

File: frontend/src/components/Home.css [hunk 4]
```diff
@@ -41,59 +41,66 @@
...
   text-decoration: none;
-  color: black;
-  background-color: rgba(70, 170, 221, 0.42);
-  margin: 0.4em;
-  padding: 0.5em;
-  border-radius: 0.25em;
-  transition: 0.1s all ease-in-out;
+  color: #000000;
+  background-color: rgba(255,255,255,.8);
+  border-bottom: 2px solid rgba(0,0,0,.2);
```
Comment (Harjot1Singh) [inline frontend/src/components/Home.css]:
```suggestion
  border-bottom: 2px solid rgba(0, 0, 0, 0.2);
```

File: frontend/src/components/Home.css (hunk 3 above)
Comment (Harjot1Singh) [reply frontend/src/components/Home.css]: Or even:
```suggestion
  border-bottom-color: #000000;
```

File: frontend/src/components/Home.css [hunk 5]
```diff
@@ -41,59 +41,66 @@
...
+  width: auto;
   text-decoration: none;
-  color: black;
-  background-color: rgba(70, 170, 221, 0.42);
-  margin: 0.4em;
-  padding: 0.5em;
-  border-radius: 0.25em;
-  transition: 0.1s all ease-in-out;
+  color: #000000;
+  background-color: rgba(255,255,255,.8);
```
Comment (Harjot1Singh) [inline frontend/src/components/Home.css]:
```suggestion
  background-color: rgba(255, 255, 255, 0.8);
```

File: frontend/src/components/Home.css [hunk 6]
```diff
@@ -41,59 +41,66 @@
...
+  border-bottom: 2px solid rgba(0,0,0,.2);
+  text-align: center;
+  transition: 0.125s all ease-in-out;
 }
 
 .home .sources .source:hover {
-  font-family: 'Open Gurbani Akhar Black';
-  background-color: rgba(70, 170, 221, 0.7);
+  background: #DCCDA2;
+  border-bottom: 2px solid rgba(0,0,0,.1);
```
Comment (Harjot1Singh) [inline frontend/src/components/Home.css]:
```suggestion
  border-bottom: 2px solid rgba(0, 0, 0, 0.1);
```

File: frontend/src/components/Home.css [hunk 7]
```diff
@@ -41,59 +41,66 @@
...
-@media screen and (max-width: 768px) {
-  .home {
-    font-size: 2.7vw;
+@media screen and (max-width: 1100px) {
+  .home .introduction, .home .sources {
+    width: 100%
   }
-
-  .home section {
+  .home .introduction {
```
Comment (Harjot1Singh) [inline frontend/src/components/Home.css]: Missing newline

File: frontend/src/components/Home.css [hunk 8]
```diff
@@ -41,59 +41,66 @@
...
     width: 100%;
+    position: static;
+    min-height: 50vh;
   }
-
```
Comment (Harjot1Singh) [inline frontend/src/components/Home.css]: Missing newline

File: frontend/src/components/SourcePage.css [hunk 9]
```diff
@@ -1,62 +1,66 @@
...
-  padding: 0.1em;
-  line-height: 1.75em;
-  border-radius: 0.1em;
-  margin-right: 0.2em;
-  transition: 0.1s all ease-in-out;
-}
-
-.source-page .lines .line:last-child {
-  margin-right: 0;
+  padding: 0.18em;
```
Comment (Harjot1Singh) [inline frontend/src/components/SourcePage.css]: Repeated property

File: frontend/src/components/SourcePage.css [hunk 10]
```diff
@@ -1,62 +1,66 @@
...
+  line-height: 1.756em;
+  border-radius: 0.18em;
+  transition: 0.125s all ease-in-out;
+  padding: .18em .238em;
```
Comment (Harjot1Singh) [inline frontend/src/components/SourcePage.css]:
```suggestion
  padding: 0.18em 0.238em;
```

File: frontend/src/components/SourcePage.css [hunk 11]
```diff
@@ -1,62 +1,66 @@
...
+  border-radius: 0.136em;
 }
 
 .source-page .lines .line:hover, .source-page .lines .line:focus {
   cursor: pointer;
-  background-color: rgba(70, 170, 221, 0.42);
   outline: none;
+  border-bottom: 2px solid rgba(0,0,0,.1);
```
Comment (Harjot1Singh) [inline frontend/src/components/SourcePage.css]:
```suggestion
  border-bottom: 2px solid rgba(0, 0, 0, 0.1);
```

File: frontend/src/components/SourcePage.css [hunk 12]
```diff
@@ -1,62 +1,66 @@
...
+}
+
+.source-page .lines .line:hover {
+  background: #DCCDA2;
+  border-bottom: 2px solid rgba(0,0,0,.1);
```
Comment (Harjot1Singh) [inline frontend/src/components/SourcePage.css]:
```suggestion
  border-bottom: 2px solid rgba(0, 0, 0, 0.1);
```

File: frontend/src/components/SourcePage.css [hunk 13]
```diff
@@ -1,62 +1,66 @@
...
+}
+
+.source-page .lines .line:focus {
+  background: #F5CD3D;
+  border-bottom: 2px solid rgba(0,0,0,.1);
```
Comment (Harjot1Singh) [inline frontend/src/components/SourcePage.css]:
```suggestion
  border-bottom: 2px solid rgba(0, 0, 0, 0.1);
```

File: frontend/src/components/SourcePage.css [hunk 14]
```diff
@@ -1,62 +1,66 @@
...
 
 .source-page .controls .right.button {
   margin-left: auto;
 }
-
-@media screen and (min-width: 1200px) {
-  .source-page {
-    font-size: 12px;
-  }
-}
```
Comment (Harjot1Singh) [inline frontend/src/components/SourcePage.css]: Missing EOF newline

File: frontend/src/components/Home.css (hunk 2 above)
Comment (bhajneet) [reply frontend/src/components/Home.css]: 83.33 *1.2 = ~100

File: frontend/src/components/SourcePage.css (hunk 10 above)
Comment (bhajneet) [reply frontend/src/components/SourcePage.css]: lint: add preceding 0 to decimal values

File: frontend/src/components/Home.css (hunk 3 above)
Comment (Harjot1Singh) [reply frontend/src/components/Home.css]: That's fine, just correct the spacing in `rgba` in that case

**Pull request merged**